explain/index.mdx

```diff
@@ -51,5 +51,5 @@ pganalyze automatically analyzes EXPLAIN plans to find the most important insigh
 * **[Stale Stats](/docs/explain/insights/stale-stats)**<br />The table referenced has not had an `ANALYZE` run recently (potentially leading to inefficient plans)
 
 <br />
-<div><feature-page-cta event-label="Docs" title="Automatic Insights into Postgres Query Plans" description="pganalyze automatically gathers the auto_explain output on your behalf from the Postgres logs, so you can easily view the plans for slow queries without extra effort." path="/postgres-explain"></feature-page-cta></div>
+<div><FeaturePageCTA eventLabel="Docs" title="Automatic Insights into Postgres Query Plans" description="pganalyze automatically gathers the auto_explain output on your behalf from the Postgres logs, so you can easily view the plans for slow queries without extra effort." path="/postgres-explain" /></div>
 <br />
```

Comment:
It looks like the EXPLAIN index page broke, possibly due to the image changes requiring `visualization.png` to be included via an import instead of inline? (I don't think the CTA change is causing it)

<img width="2204" alt="Screen Shot 2021-01-16 at 2 25 41 PM" src="https://user-images.githubusercontent.com/7227/104824464-b7b3e400-5806-11eb-9f64-ce4c3ade4598.png">

Reply:
Yeah, that was it. Changed it to use an import. It looks like using relative paths directly for images works with markdown images (`![](...)`), but not JSX images (`<img ... />`). The latter needs to use imported paths (which also breaks the gatsby-remark image optimization and bells and whistels, but again not a big deal in this case).